source/test/main.test.ts

```diff
@@ -71,4 +71,11 @@ test("has output - bucketName", () => {
     const stack = new MainStack(app, 'test');
     const template = Template.fromStack(stack);
     template.hasOutput('bucketName', {})
+})
+
+test("has - 1 CustomResource", () => {
```

Comment:
This test case is low quality. It just makes sure we create a Custom Resource, but it does not verify any detail of it. Something important factors should be verified in test case, for example, Lambda of custom resource is granted the necessary and least privilege, vpc, sg configuration and so on.

Reply:
add one more test case.

I think the test for CDK is to make sure that the ClouldFormation template is generated  as expected.  The factors you mentioned are very important,  they are already covered by cfn-nag check. 

I referred to the CDK document to write these test cases: 
https://docs.aws.amazon.com/cdk/v2/guide/testing.html